CMakeLists.txt

```diff
@@ -36,7 +38,7 @@ list ( APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/" )
 find_package( Julia REQUIRED )
 
 # Find t8code
-find_package( T8code )
+find_package( T8CODE CONFIG )
```

Comment:
Why `CONFIG`?

Reply:
Short answer: you get annoying warnings else

Long answer: there are two modes in which `find_package` operates: https://cmake.org/cmake/help/latest/command/find_package.html#search-modes
"module mode" is the default one and will first try to find a `FindT8code.cmake` file. T8code however provides `T8CODEConfig.cmake` and this is only found by "config mode".

Reply:
OK, thanks. Is this maybe an issue that should be reported upstream, since it seems to be non-standard and confusing (at least to me 😬).

Reply:
Actually not t8code's fault. The CMake documentation is pretty clear on that:
> The Find<PackageName>.cmake file is not typically provided by the package itself. Rather, it is normally provided by something external to the package, such as the operating system, CMake itself, or even the project from which the find_package() command was called. Being externally provided, [Find Modules](https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#find-modules) tend to be heuristic in nature and are susceptible to becoming out-of-date. They typically search for certain libraries, files and other package artifacts.

So, t8code does not support `module` mode per default. This is up to the system admins offering t8code as a module.

Reply:
So, I would say, `find_package( T8CODE CONFIG )` is the way to go to avoid any superfluous warnings.

Reply:
TIL! Thanks for the explanation!